Remotion/Data/DomainObjects/Infrastructure/DomainObjectTransactionContext.cs

```diff
@@ -21,48 +21,37 @@
 namespace Remotion.Data.DomainObjects.Infrastructure
 {
   /// <summary>
-  /// Provides the default implementation of the <see cref="IDomainObjectTransactionContext"/> interface.
+  /// Provides actual implementation for the <see cref="DomainObjectTransactionContextStruct"/>.
   /// Represents the context of a <see cref="DomainObject"/> that is associated with a specific <see cref="ClientTransaction"/>.
   /// </summary>
-  public class DomainObjectTransactionContext : IDomainObjectTransactionContext
+  public class DomainObjectTransactionContext
```

Comment:
Since we call it strategy in a few places we might want to rename this to have a `Strategy` suffix. This would then match a bit better with its intent.